app/server/models/user.py

```diff
@@ -425,7 +425,7 @@ def users_within_radius_filter(self, radius):
 
     def get_transfer_account_for_organisation(self, organisation):
         for ta in self.transfer_accounts:
-            if ta in organisation.transfer_accounts:
+            if ta.organisation.id == organisation.id:
```

Comment:
presumably something was breaking about the old check?

Reply:
I just thought it might be faster checking for an ID match than a list membership check